package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Custom jest matchers to test the state of the DOM",
   "main": "dist/index.js",
   "engines": {
-    "node": ">=8",
-    "npm": ">=6"
+    "node": ">=10.13",
+    "npm": ">=6",
+    "yarn": ">=1"
```

Comment:
These changes are the only reason this would be a breaking change, but they're not necessary to upgrade all the dependencies.

I'd recommend reverting this so we can upgrade the tooling without shipping a breaking change.

Reply:
I updated these too, because `kcd-scripts` needs `Node 10.13+`, so the tests can only run on 10.13+ now.

Reply:
But `kcd-scripts` is used for development only while the `engines` field is checked for production installs. As Kent said: We can upgrade the deps without ever touching the `engines` field.

Reply:
Yeah I understand, but we would still lose the possibility to test against Node 8 from now on
So any breaking change won't be caught.

Reply:
Sure. We don't automatically test for a lot of things. That doesn't mean we can't support them or have to restrict library consumers from using this library in an unsupported way.

Reply:
True that 🙂

I'll revert the `engines` to support Node 8 again. 🙂